Controller/AdminSecurityController.php

```diff
@@ -74,7 +74,7 @@ public function loginAction(Request $request = null)
 
         // TODO: Deprecated in 2.3, to be removed in 3.0
         try {
-            $resetRoute = $this->generateUrl('sonata_user_admin_resetting_request');
+            $resetRoute = $this->generateUrl('sonata_user_resetting_request');
```

Comment:
Not sure, if this is the right route. Do we have a special one for the admin bundle?